elementary/config/config.py

```diff
@@ -91,3 +93,7 @@ def _validate_elementary_profile(self):
                 raise NoElementaryProfileError
         except FileNotFoundError:
             raise NoProfilesFileError(self.profiles_dir)
+
+    def validate_timezone(self):
```

Comment:
We need to call this function.
Could you please add it to `validate_monitor()` at the end?

Reply:
@elongl I add a call for the `validate_timezone` from the `validate_monitor()`, is this the correct why?

Reply:
Yeah, looks great.